src/styles/Book.css

```diff
@@ -1,3 +1,40 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka+One&family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+.con-books {
+  min-width: 18rem;
+  height: 10.625rem;
+  margin: 2.313rem 8%;
+  padding: 2rem 0 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+}
+
+.info-book {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100%;
+}
+
+.info-book h2 {
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'RobotoSlab', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.info-book h3 {
+  width: auto;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 0.875rem;
+  font-weight: bold;
+}
+
```

Comment:
- [ ] One of the requirements says that all elements in design should be added - Features that haven’t been implemented yet should be added as a placeholder and do not need to be implemented. kindly make sure you add all missed sections in your Book component like (progress bar, ....)
- [ ] Also make sure you use the same background-color and padding as the original design state. check here the difference between your design and the template one :point_down: 

Your design | the template
---|---
![image](https://user-images.githubusercontent.com/65068771/155482076-b7eda702-4d60-4d6a-99e3-46fb697eef60.png) | ![image](https://user-images.githubusercontent.com/65068771/155482207-7948a65e-2c91-41de-a673-0c7118464c9c.png)